brave/src/main/java/brave/propagation/StrictScopeDecorator.java

```diff
@@ -30,38 +35,66 @@
  *                  ).build();
  * }</pre>
  */
-public final class StrictScopeDecorator implements ScopeDecorator {
-  public static ScopeDecorator create() {
+// Closeable so things like Spring will automatically execute it on shutdown and expose leaks!
```

Comment:
This is a neat trick.. Let's say some instrumentation you didn't write might be leaking a scope. If you add a bean of this type to an app with sleuth.. when you ctrl-c it *should* expose any leak problem automatically.